scripts/build-manifest.sh

```
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+ROOT_DIR=$(
+  cd $(dirname "$0")/.. || exit
+  pwd
+)
+
+ARTIFACT_ROOT="${ROOT_DIR}/artifacts"
+
+build() {
+  local manifest_dir=$1
+  local overlays="$(echo "${manifest_dir}" | sed 's!manifests/!!')"
+  local artifact="manifest.yaml"
+
+  set -e
```

[Question] Why `set -e` only here? 

I feel enabling `-e` whole shell is more better for avoid hiding errors.
You may also add `-u`.

---

Because `diff` command returns exit code `1` when there're any differences.
But, yeah, it makes sense to use `set -u` in the entire script. I'll add it.

---

Oh, I see!

[Tips] We can suppress `diff`'s exit code like below.
```bash
git diff --quiet || true
```

---

Not `git diff`, but `diff` shell command. So, I've updated like this `diff ... || :`.